.github/workflows/ci.yml

```diff
@@ -175,3 +182,38 @@ jobs:
           version: "v${{ env.VERSION }}"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  label:
+    name: Attach automerge label
+    # only for PRs by softwaremill-ci
+    if: github.event.pull_request.user.login == 'softwaremill-ci'
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      # count number of files changed
+      - name: Count number of files changed
+        id: count-changed-files
+        run: |
+               N=$(git diff --name-only -r HEAD^1 HEAD | wc -w)
+               echo "changed_files_num=$N" >> $GITHUB_OUTPUT
+      - name: Launch labeler
+        # skip if more than one file changed
+        if: steps.count-changed-files.outputs.changed_files_num == 1
```

Comment:
I can't see a scenario in which this "modified file swap" thing could happen - would you pls give an example?

Reply:
Apparently I don't understand something. Doesn't it happen when I modify 1 file - and it's some other file than expected?

Reply:
There is white list of three files that we consider valid for "automerge" in file `labeler.yml`. So if steward modifies <> 1 file nothing happens, if it modifies exactly 1 one, then that file will be checked against that white list - if it is on it, then `automerge` label will attached, otherwise no label.

Reply:
Added.